app/cafe.py

```diff
@@ -0,0 +1,21 @@
+from datetime import date
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if visitor.get("vaccine") is None:
+            raise NotVaccinatedError("The visitor is not vaccinated!")
+        elif visitor.get("vaccine").get("expiration_date") < date.today():
```

Comment:
Modify this line to prevent calling `get` from `None`
```suggestion
        elif visitor.get("vaccine", {}).get("expiration_date") < date.today():
```